core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -537,6 +538,26 @@ public void removeBlock(long sessionId, long blockId)
     mLocalBlockStore.removeBlock(sessionId, blockId);
   }
 
+  @Override
+  public void removeAllBlock(long sessionId)
+      throws InvalidWorkerStateException, IOException {
+    BlockStoreMeta mBlockStoreMeta = getStoreMetaFull();
+    for (List<Long> mBlockIds : mBlockStoreMeta.getBlockList().values()) {
+      for (long mBlockId: mBlockIds) {
+        try {
+          mLocalBlockStore.removeBlock(sessionId, mBlockId);
+        } catch (BlockDoesNotExistException e) {
+          LOG.info("No such block id: " + mBlockId);
+        }
+      }
+    }
+  }
+
+  @Override
+  public void tryRemoveAllBlock(long sessionId) throws IOException {
+    mLocalBlockStore.tryFreeAllSpace(sessionId);
```

Comment:
why we have to use two api method here, can we use only one method with a removeAllBlockOption to specify how to free.